tests/unit/conftest.py

```diff
@@ -39,12 +37,5 @@ def pyramid_request(
     pyramid_config,  # pylint:disable=unused-argument
 ):
     pyramid_request = testing.DummyRequest()
-
     apply_request_extensions(pyramid_request)
-
-    pyramid_request.checkmate = create_autospec(
-        ViaCheckmateClient, spec_set=True, instance=True
-    )
-    pyramid_request.checkmate.check_url.return_value = None
```

Comment:
We no longer need `request.checkmate` in the tests, since it isn't used/doesn't exist anymore